### DISCUSSION TITLE
Thank you for this project; GNOME spin is probably the best distro I used since Linux Mint

### DISCUSSION THREAD

**Discussion by banzaigtv** [SpiralLinux/SpiralLinux-project#276] [General]
I recently installed Spiral Linux with the GNOME desktop on an external SSD. I was pleased that it took Debian 11 and made it compatible with my 11th-gen Intel hardware. In fact, it was good enough to be promoted to being my HP laptop's main boot OS, so I transferred it to the system's internal NVMe SSD. After testing out a few desktop versions, I found the GNOME spin to be the most well-done edition of Spiral Linux and should be considered the flagship desktop environment for the project.

I have the laptop connected to a 4K external display. I tried out various distros with different desktop environments. I've been mainly a KDE Plasma user for the last three years, but found that the Plasma desktop has an issue where the display scaling value is always duplicated on both my laptop display and my 4K display when using X11. Wayland on Plasma 5.27 addresses that issue, but some applications do not work too well on Wayland. Cinnamon has frame rate issues when 200% scaling is used. XFCE scales some applications poorly and Budgie was a mess. Finally, GNOME 3.38 fixes all of the above issues, even on X11, while being fast, stable, and low on CPU resources. I tried GNOME 43 on Debian 12, but it was unstable when extensions are used.

Overall, I think the GNOME spin of Spiral Linux is a job well done. While the KDE Plasma and Budgie versions need work, I'm not too shy about sticking with the legacy GNOME 3 desktop for my needs. But when Debian 11 "Bullseye" reaches end of support in 2026, I would be forced to retire from GNOME 3.38 and find something new. I would suggest including the Yaru Colors theme set because it looks nice, clean, and simple. I would also want to see a fork of GNOME 3.38 made into a standalone desktop environment as a spiritual successor to the MATE desktop. If I encounter any issues, then I'll report them here.

**Comment by geckolinux** [comment]
Hi there, thanks a lot for the complements, I'm really glad to hear it's working well for you.

> While the KDE Plasma and Budgie versions need work

True, although SpiralLinux is not a software development project, and it strictly uses unmodified Debian packages with no external repositories. Every attempt is made to improve the SpiralLinux default experience as much as permitted by the configuration of the underlying system and the desktop environment. So most limitations in the KDE Plasma and Budgie spins can not be improved any further. However it would be worth testing both editions after upgrading them to Debian 12 to see if the upstream development has improved any aspects (especially fundamental functionality like screen scaling).

> Cinnamon has frame rate issues when 200% scaling is used.

I would recommend that you leave the scaling factor untouched and try increasing the "text scaling factor" (DPI) setting:

![Screenshot from 2023-06-24 09-29-35](https://github.com/SpiralLinux/SpiralLinux-project/assets/2692138/506a5748-9a97-475a-8305-571ca95c2f6f)

> I would suggest including the Yaru Colors theme set because it looks nice, clean, and simple

Agreed, but it wasn't available in the Debian repos for Debian 11 and therefore couldn't be included. However it looks like it does exist for Debian 12, so we can take a look at it for the next SpiralLinux release. No guarantees though; a lot of themes aren't viable because they don't work well with the different panels and menus in some of the desktop environments that SpiralLinux targets.

> I would also want to see a fork of GNOME 3.38 made into a standalone desktop environment as a spiritual successor to the MATE desktop.

That definitely wouldn't come out of the SpiralLinux project, as I have a strict policy of no custom packages and no software maintenance load. Fortunately what you suggest already exists, it's called Cinnamon. ;-)

Thanks again for your comments.

**Comment by banzaigtv** [comment]
I installed the Nemo file manager and set it as default, then added the Nemo script to set image as wallpaper on the GNOME desktop. Oh, man, this makes the overall experience even better! Nemo is much better than Nautilus.

**Comment by mundey** [comment]
xfce is my DE, due to simplicity of work. I don't tweak much in DE - change to dark theme and don't show top bar when window is maximized. The best use of screen :)

![spiral-xfce](https://github.com/SpiralLinux/SpiralLinux-project/assets/46802402/e6c93837-ce94-44a7-a67a-00fef9160e65)

**Comment by bunnymnemonic** [comment]
I would like to say thank you for this distro as well. It works out of the box. It is very well crafted. It is what I was looking for (I tried the XFCE spin). Thank you for maintaining it!

**Reply by geckolinux** [reply]
Really glad it's working well for you, thanks for saying so!
